src/libteam/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+## This script is to build libteam
+##
+## USAGE:
+##   ./build.sh
+
+# Obtain libteam
+rm -rf ./libteam
+git clone https://github.com/jpirko/libteam.git
+pushd ./libteam
+git checkout -f 8c1cea0
+popd
+
+git clone https://anonscm.debian.org/git/collab-maint/libteam.git tmp
+pushd ./tmp
+git checkout -f da006f2
```

Comment:
I will prefer to use tag. I notice using uscan does the same job except that I need to do more untar and folder moving jobs before making the deb packages.